packages/alfa-rules/src/common/expectation/contrast.ts

```diff
@@ -223,3 +230,42 @@ export function contrast(foreground: RGB, background: RGB): number {
 
   return round(contrast * 100) / 100;
 }
+
+/**
+ * Finds elements from a collection of candidate that intersect with a given element
+ *
+ * @remarks
+ * If `candidates` is non-empty and just one element doesn't have layout `None` is returned
+ */
+function getIntersectors(
+  element: Element<string>,
+  candidates: Iterable<Element>,
+  device: Device
+): Option<Iterable<Element>> {
+  // If the collection of candidates is empty we don't need layout to determine that there are no intersectors
+  if (Iterable.isEmpty(candidates)) {
+    return Option.of(candidates);
+  }
+
+  const elementBox = getBoundingBox(element, device);
+
+  // If just one of the elements involved doesn't have a bounding box, we regard that as an invalid state for deciding intersectors and return `None`
```

Comment:
```suggestion
```
I feel we do not need to duplicate the comment since it's already part of the TSdoc.